spec/features/course/assessment/submission/guided_spec.rb

```diff
@@ -53,6 +48,15 @@
         expect(page).to have_selector('h2', text: mrq_questions.first.title)
 
         click_link '2'
+        # Students should not be able to go to Q2 before Q1 is done.
```

Comment:
For performance reasons, I lean to merge them in feature spec. There were some discussions here: https://github.com/Coursemology/coursemology2/pull/1172